replicaset.go

```diff
@@ -66,25 +61,44 @@ func Initiate(session *mgo.Session, address, name string, tags map[string]string
 	monotonicSession := session.Clone()
 	defer monotonicSession.Close()
 	monotonicSession.SetMode(mgo.Monotonic, true)
-	cfg := Config{
-		Name:    name,
-		Version: 1,
-		Members: []Member{{
-			Id:      1,
-			Address: fixIpv6Address(address),
-			Tags:    tags,
-		}},
+	// We don't know the mongod's ability to use a correct IPv6 addr format
+	// until the server is started, but we need to know before we can start
+	// it. Try the older, incorrect format if the correct format fails.
+	cfg := []Config{
```

Comment:
configs ? something plural is probably useful since its now a list.

I also wonder if we really want this, or if it would be better to just always use the proper IPv6 form, and drop support for 2.4 + IPv6.